openssl-static/pom.xml

```diff
@@ -35,6 +35,9 @@
     <linkStatic>true</linkStatic>
     <!-- Don't deploy this artifact to Maven Central -->
     <maven.deploy.skip>true</maven.deploy.skip>
+
+    <!-- Skip tests for now as there is something wrong with the static build for openssl when running on macOS -->
+    <skipTests>true</skipTests>
```

Comment:
Can be removed once https://github.com/netty/netty-tcnative/pull/261 is merged :)